<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comfortaa</family>
     </font>
    </property>
    <property name="text">
     <string>add back</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="enterEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comfortaa</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="dequeInfoLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>331</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comfortaa</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="dequeLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>259</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comfortaa</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="frontButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comfortaa</family>
     </font>
    </property>
    <property name="text">
     <string>add front</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comfortaa</family>
     </font>
    </property>
    <property name="text">
     <string>show</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="enterEdit_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comfortaa</family>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="frontButton_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>80</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comfortaa</family>
     </font>
    </property>
    <property name="text">
     <string>add front</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comfortaa</family>
     </font>
    </property>
    <property name="text">
     <string>add back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showButton_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>80</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comfortaa</family>
     </font>
    </property>
    <property name="text">
     <string>show</string>
    </property>
   </widget>
   <widget class="QLabel" name="dequeLabel_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>259</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comfortaa</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="dequeInfoLabel_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>160</y>
      <width>331</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comfortaa</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="findEdit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comfortaa</family>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="findButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comfortaa</family>
     </font>
    </property>
    <property name="text">
     <string>find</string>
    </property>
   </widget>
   <widget class="QPushButton" name="findButton_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>80</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comfortaa</family>
     </font>
    </property>
    <property name="text">
     <string>find</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="findEdit_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comfortaa</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="findLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>120</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comfortaa</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="findLabel_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>110</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comfortaa</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="mergeButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>360</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comfortaa</family>
     </font>
    </property>
    <property name="text">
     <string>merge</string>
    </property>
   </widget>
   <widget class="QLabel" name="dequeMergeLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>400</y>
      <width>571</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comfortaa</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
